lib/Tomcat/Utils.pm

```diff
@@ -26,15 +26,14 @@ use constant TIMEOUT => 60;
 
 # Use keyboard to browse the examples faster
 sub browse_with_keyboard {
-    my ($self, $fallback_needle, $test_func, $tab_num) = @_;
+    my ($self, $fallback_needle, $test_func, $tab_num, $mod_jk) = @_;
 
     assert_screen($fallback_needle, TIMEOUT);
     for (1 .. $tab_num) { send_key('tab'); }
     send_key('ctrl-ret');
     send_key('ctrl-tab');
 
-    $test_func->();
-
+    $test_func->($mod_jk);
```

Comment:
Do we need to pass`$mod_jk` here as an argument? I think all the `$test_func` are independent of `mod_jk`being set